graph/appender/dead_node.go

```diff
@@ -27,11 +26,6 @@ func (a DeadNodeAppender) AppendGraph(trafficMap graph.TrafficMap, globalInfo *G
 		return
 	}
 
-	var err error
```

Comment:
Is this safe ?
I get that the initialization of globalInfo.Business happens in a different area and this "nil" condition won't happen again ?